From ad5a49e518724349b97311c661b6bab55efd0b7c Mon Sep 17 00:00:00 2001
From: Michael Prokop <mika@debian.org>
Date: Wed, 19 May 2010 16:59:24 +0200
Subject: [PATCH] Support booting from USB 3 xHCI-based controllers (using kernel driver xhci).

---
 hook-functions |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hook-functions b/hook-functions
index 2f6580b..b486439 100644
--- a/hook-functions
+++ b/hook-functions
@@ -364,7 +364,7 @@ auto_add_modules()
 {
 	case "${1:-}" in
 	base)
-		for x in ehci-hcd ohci-hcd uhci-hcd usbhid hid-apple btrfs \
+		for x in ehci-hcd ohci-hcd uhci-hcd usbhid xhci hid-apple btrfs \
 		ext2 ext3 ext4 ext4dev isofs jfs nfs reiserfs udf xfs \
 		af_packet atkbd i8042 virtio_pci; do
 			manual_add_modules "${x}"
-- 
1.7.1

